Add missing ABI compatibility check for return parameters in inherited functions (#10004)
ethereum/solidity: docs/layout-of-source-files.rst

the same pragma, even though it is not considered experimental
anymore.

.. note::

>   This pragma applies to all the code defined in the file where it's activated,

Reviewer: ```suggestion
  This pragma applies to all the code defined in the file where it is activated,
```

I think we don't use contractions in docs.

Author: Done.

>   regardless of where that code ends up eventually. This means that a contract
>   not explicitly using the new encoder can still contain code that uses it by
>   inheriting it from another contract. This is allowed if the new types are only

Reviewer: ```suggestion
  regardless of where that code ends up eventually. This means that a contract
  without the `ABIEncoderV2` pragma can still contain code that uses it by
  inheriting it from another contract. This is allowed if the new types are only
```
or something similar

Author: Yeah. Just trying to find ways not to say `ABIEncoderV2` all the time :)

Author: Done.
